app/kubernetes/views/deploy/deploy.html

```diff
@@ -40,6 +28,19 @@
                     <span class="ml-2 mb-1 switch-values" ng-if="!ctrl.formValues.namespace_toggle">No</span> -->
                   </div>
                 </div>
+
```

Comment:
remove these empty lines to match the EE pr, it's not a big deal, but could make applying future patches between projects a bit faster